client/components/Byline/Byline.tsx

```diff
@@ -50,6 +58,45 @@ const Byline = (props: PrivateBylineProps) => {
 		truncateAt = null,
 	} = props;
 
+	let bylinePrefixObject;
+
+	switch (bylinePrefix) {
+		case BylineOption.By:
+			bylinePrefixObject = BylineOption.By;
+			break;
+		case BylineOption.Contributors:
```

Comment:
If the `BylineOption -> IconName` mapping were stored in an object somewhere nearby:

```ts
const iconsForBylineOptions: Record<BylineOption, IconName> = {
    [BylineOption.Contributors]: 'people',
    // ...
}
```
...then we could collapse these three cases into the same piece of code, since they otherwise do the same thing.